evap/staff/importers.py

```diff
@@ -705,8 +744,17 @@ def create_test_success_messages(self):
         filtered_users = [user_data for user_data in self.users.values() if not user_data.user_already_exists()]
 
         self.success_messages.append(_("The test run showed no errors. No data was imported yet."))
-        msg = format_html(_("The import run will create {} users:"), len(filtered_users))
-        msg += create_user_list_html_string_for_message(filtered_users)
+        if len(filtered_users) == 0:
+            msg = format_html(_("The import run will create no users."))
+        else:
+            msg = format_html(
+                ngettext(
+                    "The import run will create {user_count} user:",
+                    "The import run will create {user_count} users:",
+                    len(filtered_users),
+                ).format(user_count=len(filtered_users))
+            )
```

Comment:
This `.format` for the first argument of `format_html` is an anti-pattern here and a vulnerability in other places. As a rule of thumb, you should _never_ use `format_html` with only one argument. In your case here, you could write
```python
msg = format_html(
    _("Successfully created {count} {object}:{list}"),
    count=len(created_users),
    object=ngettext("user", "users", len(created_users)),
    list=create_user_list_html_string_for_message(created_users),
)
```